recipe-server/client/control_new/components/common/NavigationMenu.js

```diff
@@ -18,17 +18,17 @@ export function NavigationMenu({ router }) {
       <Item key="/"><Link href="/">Home</Link></Item>
 
       <SubMenu title="Recipes" key="Recipes">
-        <Item key="/recipes">
-          <Link href="/recipes">View All</Link>
+        <Item key="/recipe">
```

Comment:
I thought we had settled during the workweek on singular nouns since that's what the API already uses for the URLs backing these views. That's the main reason I've been changing them around, I think we should be consistent across our URLs one way or the other.